src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -327,7 +420,7 @@ const theme = {
 
   alert,
   badge,
-  button,
+  button: generateButtonTheme(aliases),
```

Comment:
This is really cool, but I'm not sure I understand what you're doing very well ... Please, how will you identify if the user has selected a night/light mode and pass this information here? how the aliases with dark colors look like?

Reply:
Note that this is only a step towards dark mode - the "blueprint" for these changes are at https://github.com/getsentry/sentry/pull/20118 but it involves lots of changes so I'm trying to break them down into digestable chunks.